src/pages/Landing/index.tsx

```diff
@@ -56,6 +56,25 @@ export default () => {
   const columnHeaders = ['Name', 'Age', 'Is Cool'];
 
   if (isLoading) return <Loader />;
+  function handleDelete(customerId: any): void {
+    function deleteCustomer() {
+      throw new Error('Function not implemented.');
+    }
+
+    try {
+      deleteCustomer();
```

Comment:
if you test this, you'll see that its calling`     throw new Error('Function not implemented.');` above. which isn't what you want.  You want it to call `deleteCustomer` from CustomerApi.tsx